lib/marked.js

```diff
@@ -1217,28 +1238,20 @@ Parser.prototype.tok = function() {
     }
     case 'list_item_start': {
       body = '';
+      var loose = this.token.loose;
```

Comment:
Any reason why you create a new var here instead of using `this.token.loose` below?

Reply:
because `this.token` is changed when `this.next()` is called in the while loop below.

`body += !this.token.loose && this.token.type === 'text'` would be checking if the text token is loose not the list start token